src/securejoin.rs

```diff
@@ -625,13 +657,15 @@ pub(crate) async fn observe_securejoin_on_other_device(
                 // FIXME: Old versions of DC send this header instead of gossips. Remove this
                 // eventually.
                 let fingerprint = fingerprint.parse()?;
+                let backward_verified = true; // TODO check
```

Comment:
Maybe we already remove this compatibility code as all the versions since Jan don't need it? Instead of fixing it and adding more TODOs. F.e. the "*-received" Securejoin messages were already removed, they are not needed since the same Jan release

Reply:
Then u can also remove the code mentioned in https://github.com/deltachat/deltachat-core-rust/issues/5062 if we're not going to have this compat code anymore